starter-code/public/scripts/models/repo.js

```diff
@@ -7,11 +7,32 @@ var app = app || {};
   repos.all = [];
 
   repos.requestRepos = function(callback) {
-    // TODO: How would you like to fetch your repos? Don't forget to call the callback.
+    // DONE: How would you like to fetch your repos? Don't forget to call the callback.
     //       Remember that the callback function we'll want to call relies on repos.all
     //       being an array with a bunch of repo objects in it, so you'll need to
     //       populate it with the response from Github before you call the callback.
 
+    $.ajax({
+      url: 'https://api.github.com/user/repos',
+      method: 'GET',
+      headers: {
+        Authorization: `token ${githubToken}`
+      }
+    })
+    .then(
+        function (repos) {
+          let slimmedRepos = repos.map(repo => ({
+            name: repo.name,
+            html_url: repo.html_url,
+            description: repo.description,
+            language: repo.language,
+            created_at: repo.created_at,
+            updated_at: repo.updated_at,
+            watchers_count: repo.watchers_count
+          }));
+          callback(slimmedRepos);
```

Comment:
Why are you returning a variable that is defined inside of a function here? Don't we have a global array that we can put our data into and then use in other places?